PWGGA/GammaConv/macros/AddTask_TRDEfficiency.C

```diff
@@ -35,28 +36,39 @@ AliAnalysisTRDEfficiency* AddTask_TRDEfficiency(
     if (!mgr) {
         return 0x0;
     }
-
+    
     // ================== GetInputEventHandler =============================
     AliVEventHandler *inputHandler=mgr->GetInputEventHandler();
-    
-    // get the input event handler, again via a static method. 
-    // this handler is part of the managing system and feeds events
-    // to your task
-    if (!mgr->GetInputEventHandler()) {
+
+    //=========  Set Cutnumber for V0Reader ================================
+    TString cutnumberPhoton = photonCutNumberV0Reader.Data();
+    TString cutnumberEvent = "00000003";
```

Comment:
ok, right now you use eventcut 00000003 all the time, and I do not see that you use another one in your main task. I hope my comment was not misleading as you had another event cut before - is this what you need? otherwise just replace with the event cut that you want to run

Reply:
I think this event cut is fine for now since I specify the necessary trigger in UserExec. I'm also not sure what the effect of pileup correction will have on photon candidates in the TRD. In future I might want a dynamic event cut like one for the photons or a trainconfig system like in GammaCalo_pp. Keeping it simple might be best for now.